docs/THEOplayer/results/tizen.md

```diff
@@ -0,0 +1,50 @@
+# Test results for THEOplayer on Tizen v4.0
+
+References:
+
+- ✔️ SDK is supported.
+
+- ❌ SDK is not supported.
+
+- ![tizen](../../icons/tizen.png) Scenario supported.
+
+## VOD
+
+Preset: "AdaptiveStreaming"
+
+| Format | Clear | DRM Token | Widevine | PlayReady | FairPlay | AES-128 | Sidecar caption |
+| --------- | :---: | :---: | :----------------------------------------------------------: | :------: | :----------------------------------------------------------: | :------: | :------: |
+| HLS TS    | ❌([#5](issues.md#issue-5)) | Not  applicable  | Not applicable | Not applicable | Not applicable  | ❌([#5](issues.md#issue-5)) | ✔️ |
+| HLS CMAF  | ❌([#5](issues.md#issue-5)) | Not  applicable | Not  applicable | Not applicable | Not applicable  | ✔️ | ✔️ |
+| DASH CMAF | ✔️ | ✔️ | ✔️ | ✔️ | Not applicable | Not applicable | ✔️ |
+
+More details about issues [here](issues.md).
+
+## Live Stream
+
+EncodingType: "Standard"
+
+PresetName: "Default720p"
+
+| Format | Clear | DRM Token | Widevine | PlayReady | FairPlay | AES-128 | Live Transcription |
```

Comment:
Question on the older TIzen TV's - did we need to test the older PIFF 1.1 support that we recently added for Playready support on the 2012-2015 series TV sets? That should be a simple check, if someone has access to an old one.

Reply:
We do not support 2012-2015 series TV sets.

Reply:
2017 is supported and 2016 will be supporting in due course.